lms/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -0,0 +1,142 @@
+"""
+Tests for Blocks Views
+"""
+
+import json
+
+import ddt
+from django.test import TestCase, RequestFactory
+from unittest import skip
+
+from student.tests.factories import UserFactory
+
+from .. import adapters
+from .. import views
+
+
+@ddt.ddt
+class TestAccessTokenView(TestCase):
+    """
+    Test class for AccessTokenView
+    """
+
+    dop_adapter = adapters.DOPAdapter()
+    dot_adapter = adapters.DOTAdapter()
+
+    def setUp(self):
+        super(TestAccessTokenView, self).setUp()
+        self.user = UserFactory()
+        self.dot_app = self.dot_adapter.create_public_client(user=self.user, client_id='dot-app-client-id')
+        self.dop_client = self.dop_adapter.create_public_client(user=self.user, client_id='dop-app-client-id')
```

Comment:
Nit: code readability question - is there a reason why one variable is called `app` and the other is called `client`?